README.md

```diff
@@ -9,6 +9,10 @@ layout: default
 
 ### Check out in this [link](https://github.com/ameerhamzakhalid/capstone-module)
 
```

Comment:
- [ ] Please add the correct Github deployment live demo link instead of your repo link it will be more professional if you do that.